service_config.json

```
@@ -0,0 +1,19 @@
+{
+    "configs": [
+        {
+        "name": "oak0",
+        "port": 50010,
+        "host": "localhost",
+        "log_level": "INFO",
+        "subscriptions": [
+            {
+                "uri": {
+                    "path": "/rgb",
```

Is this accurate? Seems unused. Maybe just keeping it generic with `"*"` is better

```suggestion
                    "path": "*",
```